### PR TITLE
Instantiate new

### PR DESCRIPTION
**Pull request by Prisca-tech**

### CODE REVIEW

File: cipher.py [hunk 1]
```diff
@@ -1,18 +1,60 @@
...
+
+# to display a menu
+while (choice.lower() !="exit"):
+    print('''
+    1. Instantiate encryption parameters
+    2. Encrypt text
+    3. Decrypt text
+    4. Reset encryption parameters
+    5. History
+    ''')
```
Comment (dabensymon) [inline cipher.py:45]: Include exit to the options

File: cipher.py [hunk 2]
```diff
@@ -1,18 +1,60 @@
...
+
+    print("Enter the number that corresponds with the task you want to perform ")
+    choice = input( )
+
+    if choice == "1":
+        jumps = instantiate()
+        print(f'jumps instantiated to {jumps}')
+
```
Comment (dabensymon) [inline cipher.py:53]: include this code snippet to control the flow a bit
`       print("Press enter to continue")
        input( )
`

Comment (dabensymon) [reply cipher.py:53]: Add a else at the bottom here to give a friendly message that the choice has not been implemented yet.
eg. The choice is yet to be implemented, please check back for updates

File: cipher.py [hunk 3]
```diff
@@ -1,18 +1,60 @@
...
+    elif choice == "2":
+        encrypt()
+
```
Comment (dabensymon) [inline cipher.py:56]: Add a else at the bottom here to give a friendly message that the choice has not been implemented yet.
eg. The choice is yet to be implemented, please check back for updates

File: cipher.py [hunk 4]
```diff
@@ -1,18 +1,60 @@
+jumps = 0
+choice = " "
+
+
+
 def getName():
```
Comment (dabensymon) [inline cipher.py:6]: Unused code. Consider removing it.

**Review by dabensymon — COMMENTED** [review]
Please check and fix the comments.

**Review by Prisca-tech — COMMENTED** [review]
All suggestions has been implemented.
Thanks

File: cipher.py (hunk 2 above)
Comment (dabensymon) [reply cipher.py:55]: This is pending. Please test properly.

File: cipher.py (hunk 1 above)
Comment (dabensymon) [reply cipher.py:46]: Selecting number 6 for exit does not work

**Pull request closed without merging**